samples/common/Framework.cpp

```diff
@@ -209,11 +209,13 @@ Error SampleApp::userMainLoop(Bool& quit)
 
 		if(in.getMousePosition() != Vec2(0.0))
 		{
-			F32 angY =
-				-ROTATE_ANGLE * in.getMousePosition().x() * MOUSE_SENSITIVITY * getMainRenderer().getAspectRatio();
+			static Vec2 sEulerYawPitch = Vec2(0.0f);
```

Comment:
You mean for the initial rotation, right?. I'll modify the code so "sEulerYawPitch" is initialized with the correct angles based on the initial rotation.

Reply:
Maybe something like this could work:

```
Vec2 velocity = in.getMousePosition();
Euler angles(mover->getLocalRotation().getRotationPart());
angles.x() += velocity.y() * toRad(360.0f) * F32(elapsedTime) * MOUSE_SENSITIVITY;
angles.x() = clamp(angles.x(), toRad(-90.0f), toRad(90.0f)); // Avoid cycle in Y axis
angles.y() += -velocity.x() * toRad(360.0f) * F32(elapsedTime) * MOUSE_SENSITIVITY;
angles.z() = 0.0f;
mover->setLocalRotation(Mat3x4(angles));
```

That way you avoid the static variable. That's good since there are other types of input that can rotate the camera as well.

Reply:
LGTM! 👍 . I'll use that.